source/chapters/1/intro.md

```diff
@@ -52,33 +87,36 @@ This is the default folder structure that we will use.
 </pre>
 
 ```
-// prototype it quickly (works on Mac/Linux)
+// to create it quickly, run the commands (they should work just fine on Mac/Linux)
 mkdir -p client server imports/ui imports/startup/client imports/startup/server imports/api
 echo "import '/imports/startup/client';" > client/main.js
 echo "import '/imports/startup/server';" > server/main.js
 ```
 
 
-Everything in "/client" and "/server" is eagerly loaded (automatically loaded) [Read more](https://guide.meteor.com/structure.html#load-order) 
+Everything in "/client" and "/server" is automatically loaded, as it is explained [here.](https://guide.meteor.com/structure.html#load-order)
 
-We won't use auto-loading in order to gain more control about the file order.
+That's all great ! However, we need more control over our application, and in order to gain that control,
+we will refrain from using auto-loading in our projects.
 
-Everything in "/imports" must be explicitly imported so Meteor can know about it.
-The advantage of this is that it offers the ability to build modular applications.
-[Read more](https://danmartensen.svbtle.com/build-better-apps-with-es6-modules)
+Everything in "/imports" must be explicitly imported so Meteor can "gain knowledge" about it.
+This gives us the ability to build modular applications.
+[Read more about this here.](https://danmartensen.svbtle.com/build-better-apps-with-es6-modules)
 
 
 ## Importing from NPM
 
-For now, we won't get into much details, but the idea is that with Meteor you have access to the full NPM ecosystem, allowing you to import
+For now, we won't get into a lot of details, but the basic idea is that with Meteor you have access to the full NPM ecosystem, allowing you to import
 modules from http://www.npmjs.com
+As an idea, to get you to realise just how many resources you have at your disposal, the NPM system contained roughly 445 797 packages on May 1st of 2017.
 
 ```
-// moment is a library you will most likely use in order to manipulate dates however you want
+// "moment" is a library you will most likely use when working with dates
+// This is how you install it:
 meteor npm install --save moment
 ```
```

Comment:
Explain why we use --save:

We use save here because it will save it in our packages.json, meaning if we push and other developers do `npm install` they will have the same package with the version you specified, making life much easier. If you just want to try something out, omit the --save option.